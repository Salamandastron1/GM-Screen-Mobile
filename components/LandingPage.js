import React from 'react';
import { View } from 'react-native';
import Button from './Button';

const LandingPage = () => (
  <View style={styles.containerStyle}>
    <Button onPress={() => console.log('Ben!')}>
      Start
    </Button>
  </View>
);

const styles = {
  containerStyle: {
    flex: 1,
    flexDirection: 'row',
    padding: 5,
    alignItems: 'center',
    backgroundColor: '#f4d8b7',
  },
};

export default LandingPage;
