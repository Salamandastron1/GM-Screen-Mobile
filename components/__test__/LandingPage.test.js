/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../LandingPage';

it('should match the shapshot', () => {
  const tree = shallow(<LandingPage />);
  expect(tree).toMatchSnapshot();
});