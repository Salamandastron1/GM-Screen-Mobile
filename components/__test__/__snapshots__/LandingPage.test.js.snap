// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the shapshot 1`] = `
<MainContain>
  <Title>
    GM Screen
  </Title>
  <CardSection>
    <Component
      source={1}
      style={
        Object {
          "flex": 1,
          "height": 400,
          "width": null,
        }
      }
    />
  </CardSection>
  <CardSection>
    <Button
      onPress={[Function]}
    >
      Start
    </Button>
  </CardSection>
</MainContain>
`;
